### ISSUE TITLE
Chrome instructions (for later version)

### ISSUE THREAD

**Issue by slhenry** [w3c/EasyChecks#10]
Add instructions for doing the checks with Chrome -- and provide a filter for it per https://github.com/w3c/EasyChecks/issues/9

**Comment by slhenry** [comment]
And maybe for other tools, such as WAVE ?

**Comment by lakeen** [comment]
Suggest removing reference to Opera based on metrics numbers

**Comment by slhenry** [comment]
I agree not to have much about Opera... I went and looked and found only one reference to Opera:
"In a browser that supports keyboard navigation with the Tab key (for example, Firefox, IE, Chrome, and Safari; not Opera):"

I think since it's a caution that it doesn't work, it is good to leave it. (Also, might give them a nudge to make it work.)

**Comment by slhenry** [comment]
Consider [Accessibility Insights](https://accessibilityinsights.io/)

**Comment by slhenry** [comment]
/me looks again -- I guess above is outdated. **We already updated for Chrome** (and we list Opera):
> WebDev Toolbar - To do the checks that are indicated "with the WebDev Toolbar", you'll need either
>  * the Chrome browser and the Web Developer extension for Chrome
>  * the Opera browser and the Web Developer extension for Opera
>  * the Firefox browser and the Web Developer extension/add-on for Firefox

/me remembers now changing Firefox toolbar to WebDev toolbar to cover Chrome.

I won't close this issue, though as there are other points above.  @sharronrush

**Comment by iadawn** [comment]
Closed as no longer relevant to new version of the resource
